### PR TITLE
Command line basics translation

### PR DESCRIPTION
**Pull request by rasitds**

### CODE REVIEW

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+- [ExplainShell.com](http://explainshell.com/), özellikle garip shell komutlarının yapısını çözmek veya Bash'in nasıl çalıştığını öğrenmek istiyorsanız harika bir kaynaktır.
+- [Unix/Linux Command Cheat Sheet](https://files.fosswire.com/2007/08/fwunixref.pdf), Linux kullanımına aşina olduğunuzda düzenli olarak başvurabileceğiniz önemli komutların listesini içerir. Bilgisayarınızın başında olmadığınız zamanlarda fiziksel bir kopyasını yanınızda bulundurmak için çıktısını alabilirsiniz.
+- [Command Line Flashcards](https://flashcards.github.io/command_line/introduction.html) by flashcards.github.io.
+- [Video Series from LearnLinuxTv](https://www.youtube.com/playlist?list=PLT98CRl2KxKHaKA9-4_I38sLzK134p4GJ) komut satırının temellerini açıklayan 24 video içermektedir. Videolar yeni başlayanlar için yeterince kısa ama aynı zamanda başlamanızı sağlayacak ve içinizdeki merakı ateşleyecek kadar detaylıdır.
+
+### Bilgi Kontrolü
+
+Bu bölüm, bu dersi kendi başınıza anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayın ve bağlantı verdiği materyali gözden geçirin.
```
Comment (kutaui) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md:110]:
```suggestion
Bu bölüm, bu dersi kendi başınıza anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayın ve bağlantı verdiği materyali gözden geçirin.
```

Comment (kutaui) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
Bu bölüm, bu dersi kendi başınıza anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayın ve yönlendirdiği materyali gözden geçirin.
```

Comment (rasitds) [reply temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]: Thanks for the suggestion!

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
+### Giriş
+
+Komut satırından korkuyor musunuz? Yalnız değilsiniz. Şirketin ana bilgisayarını hacklemek için çılgınca anlaşılmaz komutlar girerken, beyaz veya yeşil metinlerin yanıp söndüğü siyah bir ekrana dikkatle bakan geliştiricilerin bu görüntüsüne sahibiz.
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
Komut satırından korkuyor musunuz? Yalnız değilsiniz. Aklımızda, şirketin ana bilgisayarını hacklemek için çılgınca anlaşılmaz komutlar girerken, beyaz veya yeşil metinlerin yanıp söndüğü siyah bir ekrana dikkatle bakan geliştiricilerin imajı canlanıyor.
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+Bu siyah ekran veya pencere, bilgisayarınızın sizin için çalıştıracağı komutları girebileceğiniz <span id="command-line">komut satırı arayüzüdür (CLI)</span>. Komut satırıyla çalışmak geliştiriciler için öğrenilmesi önemli bir beceridir. Komut satırı, diğer programları başlatabileceğimiz ve onlarla iletişime geçebileceğimiz operasyon üssü gibidir. Öğrenilmesi gereken kendine has bir sözdizimi vardır, ancak aynı komutları onlarca kez gireceğiniz için ihtiyacınız olan komutları çabuk öğrenebileceksiniz.
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
Bu siyah ekran veya pencere, bilgisayarınızın sizin için çalıştıracağı komutları girebileceğiniz <span id="command-line">komut satırı arayüzüdür (CLI)</span>. Komut satırıyla çalışmak geliştiriciler için öğrenilmesi önemli bir beceridir. Komut satırı, diğer programları başlatabileceğimiz ve onlarla iletişime geçebileceğimiz operasyon üssü gibidir. Öğrenilmesi gereken kendine has bir sözdizimi vardır ancak aynı komutları onlarca kez gireceğiniz için ihtiyacınız olan komutları çabuk öğrenebileceksiniz.
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+Komut satırına giriş niteliğindeki bu derste, bilgisayarınızda nasıl gezineceğinizi ve doğrudan komut satırının rahatlığıyla dosya ve dizinleri (klasörler olarak da bilinir) nasıl değiştireceğinizi öğreneceksiniz. Bunun sandığınız kadar zor olmadığını göreceksiniz. Bu derste öğreneceğiniz komutlar çok basittir. Bu nedenle, komut satırını ilk kez kullanmanızın sizi korkutmasına izin vermeyiniz.
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
Komut satırına giriş niteliğindeki bu derste, bilgisayarınızda nasıl gezineceğinizi ve doğrudan komut satırının rahatlığıyla dosya ve dizinleri (klasörler olarak da bilinir) nasıl değiştireceğinizi öğreneceksiniz. Bunun sandığınız kadar zor olmadığını göreceksiniz. Bu derste öğreneceğiniz komutlar çok basittir. Bu nedenle, komut satırını ilk kez kullanmak sizi korkutmasın.
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+- Bir dizini ve dosyayı yeniden adlandırmak veya yok etmek için komut satırının kullanımı.
+- Bir programda dosya veya klasör açmak için komut satırının kullanımı.
+
+### Ödev
+
+<div class="lesson-content__panel" markdown="1">
+
+**Not**: Bu kaynakların birçoğu Mac veya Linux ortamı kullandığınızı varsaymaktadır. Önceki kurulum dersimizi yaptıysanız, Linux'u çift önyükleme veya sanal bir makinede zaten kurmuş olmalısınız. Ya da MacOS kullanıyor olabilirsiniz. MacOS veya herhangi bir Linux sürümü yüklü değilse, lütfen [işletim sistemi kurulum kılavuzu](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/prerequisites) bölümüne geri dönün.
+
+1. Software Carpentry Foundation tarafından tasarlanan [The Unix Shell](https://swcarpentry.github.io/shell-novice/) kursuna göz atın. Orada CLI kullanımı ile ilgili tüm dersleri bulabilirsiniz, ancak şimdilik sadece aşağıdaki dersleri tamamlamaya odaklanalım:
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
1. Software Carpentry Foundation tarafından tasarlanan [The Unix Shell](https://swcarpentry.github.io/shell-novice/) kursuna göz atın. Orada CLI kullanımı ile ilgili tüm dersleri bulabilirsiniz ancak şimdilik, sadece aşağıdaki dersleri tamamlamaya odaklanalım:
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+   5. `test` dizininin dışına geri dönün.
+   6. `test` dizinini silin.
+
+   Bu kadar. Alıştırma ile işimiz bitti! Bundan sonra çoğu şeyi komut satırından yapmaya karar verirseniz, bu komutlar sizin için kolay ve vazgeçilmez hale gelecektir.
+
+</div>
+
+### Komut Satırını Bir Profesyonel Gibi Kullanmak
+
+Programcılar hakkında bilmeniz gereken önemli bir şey var. Programcılar bir şeyi yapmanın kolay yolunu ararlar. Gerçekten. Bir şeyi tekrar tekrar yapmaya zorlandıklarında, bunun yerine otomatikleştirmenin bir yolunu bulma ihtimalleri yüksektir. İyi haber şu ki, programcıların bu güne kadar oluşturdukları birçok kısayoldan yararlanabileceksiniz. Komut satırını bir profesyonel gibi kullanmayı öğrenmenin zamanı geldi (yani, kısayollarla kolay bir şekilde).
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
Programcılar hakkında bilmeniz gereken önemli bir şey var. Programcılar daima bir şeyi yapmanın kolay yolunu ararlar. Gerçekten. Bir şeyi tekrar tekrar yapmak zorunda kaldıklarında, büyük ihtimalle bunu otomatikleştirmenin bir yolunu bulacaklardır. İyi haber şu ki, programcıların bu güne kadar oluşturdukları birçok kısayoldan yararlanabileceksiniz. Komut satırını bir profesyonel gibi kullanmayı öğrenmenin zamanı geldi (yani, kısayollarla kolay bir şekilde).
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+### VSCode'da Komut Satırından Dosya Açmak
+
+- **Linux**: VSCode'u komut satırından `code` yazarak açabilir ve ardından konumun adını ekleyerek klasörleri veya dosyaları açabilirsiniz: `code my_awesome_project/`.
+
+- **macOS**: Bazı kurulumlar gereklidir. VSCode'u kurduktan sonra çalıştırın. VSCode çalıştıktan sonra, <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> tuşlarıyla Komut Paletini açın. Görüntülenen küçük kutuya `shell command` yazın. Görüntülenen seçeneklerden biri `Shell Command: Install 'code' command in PATH` olacaktır. Bu seçeneği seçin ve (açıksa) terminalinizi yeniden başlatın.
+
+### Ek Kaynaklar
+
+Bu bölüm ilgili içeriklere yararlı bağlantılar içerir. Bu içeriklere bakmanız zorunlu değildir, ancak ilave olarak bunlara da bakmanızı tavsiye ediyoruz.
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+
+- [The Art of Command Line](https://github.com/jlevy/the-art-of-command-line#readme) başlangıç için birebirdir!. Açık kaynak git deposudur. Burada birçok profesyonel ipucu mevcut!
+- [Learn Enough Command Line to Be Dangerous](https://www.learnenough.com/command-line-tutorial) adlı çevrimiçi kitap, komut satırında uzmanlaşmak için harika bir kaynaktır. Bölüm 1 ve 2 ücretsizdir ve komut satırı araçlarına iyi bir giriş sağlar. Kitabın geri kalanı ücretsiz değildir ve bu noktada gerçekten ihtiyacınız olandan daha fazla derinliğe iner, ancak ilginizi çekerse kitabın geri kalanını satın almaktan ve okumaktan çekinmeyin.
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Learn Enough Command Line to Be Dangerous](https://www.learnenough.com/command-line-tutorial) adlı çevrimiçi kitap, komut satırında uzmanlaşmak için harika bir kaynaktır. Bölüm 1 ve 2 ücretsizdir ve komut satırı araçlarına iyi bir giriş sağlar. Kitabın geri kalanı ücretsiz değildir ve bu noktada gerçekten ihtiyacınız olandan daha fazla derinliğe iner ancak ilginizi çekerse kitabın geri kalanını satın almaktan ve okumaktan çekinmeyin.
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+- [ExplainShell.com](http://explainshell.com/), özellikle garip shell komutlarının yapısını çözmek veya Bash'in nasıl çalıştığını öğrenmek istiyorsanız harika bir kaynaktır.
+- [Unix/Linux Command Cheat Sheet](https://files.fosswire.com/2007/08/fwunixref.pdf), Linux kullanımına aşina olduğunuzda düzenli olarak başvurabileceğiniz önemli komutların listesini içerir. Bilgisayarınızın başında olmadığınız zamanlarda fiziksel bir kopyasını yanınızda bulundurmak için çıktısını alabilirsiniz.
+- [Command Line Flashcards](https://flashcards.github.io/command_line/introduction.html) by flashcards.github.io.
+- [Video Series from LearnLinuxTv](https://www.youtube.com/playlist?list=PLT98CRl2KxKHaKA9-4_I38sLzK134p4GJ) komut satırının temellerini açıklayan 24 video içermektedir. Videolar yeni başlayanlar için yeterince kısa ama aynı zamanda başlamanızı sağlayacak ve içinizdeki merakı ateşleyecek kadar detaylıdır.
+
+### Bilgi Kontrolü
+
+Bu bölüm, bu dersi kendi başınıza anlayıp anlamadığınızı kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta zorlanıyorsanız, soruya tıklayın ve yönlendirdiği materyali gözden geçirin.
+
+- [What is the command line?](#command-line)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Komut satırı nedir?](#command-line)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+- [How do you open the command line on your computer?](#open-command-line)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Bilgisayarınızda komut satırını nasıl açarsınız?](#open-command-line)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+- [How can you navigate to a particular directory?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-cd)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Belirli bir dizine nasıl girersiniz?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-cd)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+- [Where will `cd` on its own navigate you to?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#uid31)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Tek başına `cd` sizi nereye yönlendirir?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#uid31)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 13]
```diff
@@ -0,0 +1,122 @@
...
+- [Where will `cd ..` navigate you to?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#uid30)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [`cd ..` sizi nereye yönlendirir?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#uid30)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 14]
```diff
@@ -0,0 +1,122 @@
...
+- [How do you display the name of the directory you are currently in?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-pwd)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Şu anda bulunduğunuz dizinin ismini nasıl gösterirsiniz?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-pwd)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 15]
```diff
@@ -0,0 +1,122 @@
...
+- [How do you display the contents of the directory you are currently in?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-ls)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Şu anda bulunduğunuz dizinin içeriklerini nasıl gösterirsiniz?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#sec-basics-ls)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 16]
```diff
@@ -0,0 +1,122 @@
...
+- [How do you create a new directory?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid7)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Yeni bir dizin nasıl yaratılır?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid7)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 17]
```diff
@@ -0,0 +1,122 @@
...
+- [How do you create a new file?](https://swcarpentry.github.io/shell-novice/03-create.html#create-a-text-file)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Yeni bir dosya nasıl oluşturulur?](https://swcarpentry.github.io/shell-novice/03-create.html#create-a-text-file)
```

Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Yeni bir dosya nasıl oluşturulur?](https://swcarpentry.github.io/shell-novice/03-create.html#create-a-text-file)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 18]
```diff
@@ -0,0 +1,122 @@
...
+- [How do you destroy a directory or file?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid9)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Bir dizin veya dosya nasıl silinir?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid9)
```

File: temel_bilgiler/kurulumlar/komut_satiri_temelleri.md [hunk 19]
```diff
@@ -0,0 +1,122 @@
...
+- [How do you rename a directory or file?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid10)
```
Comment (ubsoydan) [inline temel_bilgiler/kurulumlar/komut_satiri_temelleri.md]:
```suggestion
- [Bir dizin veya dosya nasıl yeniden adlandırılır?](https://www.softcover.io/read/fc6c09de/unix_commands/basics#cid10)
```

**Review by ubsoydan — CHANGES_REQUESTED** [review]
Excellent work Raşit!

CLI is the most obscure yet fun to tinker with.
May the CLI be with us!

Thanks a lot!

**Pull request merged**